Make tk85.pc work with the installed tcl85.pc.
Mimic the port and require the exact Tcl version.

Index: unix/tk.pc.in
--- unix/tk.pc.in.orig
+++ unix/tk.pc.in
@@ -9,7 +9,7 @@ Name: The Tk Toolkit
 Description: Tk is a cross-platform graphical user interface toolkit, the standard GUI not only for Tcl, but for many other dynamic languages as well.
 URL: http://www.tcl.tk/
 Version: @TK_VERSION@@TK_PATCH_LEVEL@
-Requires: tcl >= 8.5
+Requires: tcl85 = @TK_VERSION@@TK_PATCH_LEVEL@
 Libs: -L${libdir} @TK_LIB_FLAG@
 Libs.private: @XFT_LIBS@ @XLIBSW@
 Cflags: -I${includedir}
